ui_framework/components/button/button_group/button_group.test.js

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable no-undef */
```

Comment:
We need to update our eslint config with the Jest plugin, or else eslint complains about the use of `describe`, etc.